make_statistic_plots.py

```diff
@@ -0,0 +1,582 @@
+#Author: Nicole Schanche
+#Updated Oct 12, 2022
+#This script makes the TESS statistics plots for tess-website-lite. This is very much a work in progress
+
+import pandas as pd
+import sqlite3
+import json
+import re
+import numpy as np
+#import geograpy
+import itertools
+import plotly.graph_objects as go
+import plotly.express as px
+from datetime import date, datetime
+from dateutil import rrule
+import matplotlib.pyplot as plt
+from wordcloud import WordCloud
+from collections import defaultdict
+from collections import Counter
+
+
+# Get all of the data you will need to make the TESS statistics plots later on
```

Comment:
Let's make this a function so we can call it in a script easily and so we can have a clear docstring on what to expect as the outcome